cdap-ui/app/features/admin/controllers/NamespaceCreateController.js

```diff
@@ -8,10 +8,10 @@ angular.module(PKG.name + '.feature.admin')
     $scope.submitHandler = function() {
       myDataSrc.request({
         method: 'PUT',
-        _cdapPath: '/namespaces/' + $scope.model.id,
+        _cdapPath: '/namespaces/' + $scope.model.name,
         body: {
-          id: $scope.model.id,
-          name: $scope.model.name || null,
+          id: $scope.model.name,
```

Comment:
@ajainarayanan would it be ok to not send ``id`` and ``name`` in the body? It shouldn't cause an issue, since the API will ignore it, but since we're cleaning up, just checking if we can clean this too?